fs/fstest/file_windows.go

```diff
@@ -28,6 +29,13 @@ func Lchtimes(name string, atime, mtime time.Time) Applier {
 	})
 }
 
+// CreateDeviceFile provides creates devices Applier.
+func CreateDeviceFile(name string, mode os.FileMode, maj, min int) Applier {
+	return applyFn(func(root string) error {
+		return errors.New("Not implemented")
```

Comment:
Nit: should we use `errdefs.ErrNotImplemented`?

Reply:
Good catch. Let me handle it in the follow-up. (Use containerd/errdefs repo to handle this).